src/main/java/de/cas/mse/exercise/csvdb/Record.java

```diff
@@ -0,0 +1,5 @@
+package de.cas.mse.exercise.csvdb;
+
+public class Record {
+    public String[] fields;
```

Comment:
What is the benefit of this abstraction?

Reply:
I misunderstood the purpose of the exercise and thought we were supposed to make it work for representations other than CSV :|